app/Record.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace App;
+
+class Record 
+{
+    protected $table = 'records';
```

Comment:
The database table is named `amas` not `records`. Adjust this, and change the name of the model accordingly.